giraffe/src/components/geo/processing/PivotedGeoTable.ts

```diff
@@ -76,6 +77,12 @@ export class PivotedGeoTable extends AbstractGeoTable {
       this.truncated = false
     }
   }
+  mapTracks<T, U>(
```

Comment:
it might make sense to move `mapTracks` function out of `GeoTable` interface, and just have `NativeGeoTable` implement it.

Reply:
After some investigation, I realize this is needed in PivotedGeoTable. If we pass in the values for lat/lon as fields it should still render the map.

Currently, all the layers are using `GeoTable` type for props and depending on the native/pivoted that function is being executed. When we implement the TrackMaps functionality we will need to update/fix this code. Right now, it is hard coded to use the `lat` and `lon` columns for Native Geo Table. If the data doesn't include those two columns, the map fails horribly. So, for now I am leaving this as is, but creating an issue that will be put into later enhancement epics for track maps.

Reply:
Issue created to track this: https://app.zenhub.com/workspaces/applicationui-5e3b05772922e316b3a210a4/issues/influxdata/giraffe/584